src/libraries/System.Text.Json/src/System/Text/Json/Nodes/JsonNode.To.cs

```diff
@@ -45,12 +37,7 @@ public override string ToString()
                 }
             }
 
-            using var output = new PooledByteBufferWriter(JsonSerializerOptions.BufferSizeDefault);
-            using (var writer = new Utf8JsonWriter(output, new JsonWriterOptions { Indented = true }))
-            {
-                WriteTo(writer);
-            }
-
+            using PooledByteBufferWriter output = WriteToPooledBuffer(writerOptions: new JsonWriterOptions { Indented = true });
```

Comment:
It's confusing that JsonSerializerOptions is a no-no to instantiate per call, but JsonWriterOptions isn't.

Reply:
Yeah.. JsonSerializerOptions is really the only outlier, other option types in STJ behave don't hold any sort of caches.